sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/Configuration.java

```diff
@@ -61,6 +61,9 @@
     @Parameter(names = "-maxConnectionPoolSize", description = "Max Connection Pool Size")
     private Integer maxConnectionPoolSize = 1000;
 
+    @Parameter(names = "-thresholdForDiagnosticsInMs", description = "Latency threshold for printing diagnostics")
+    private Integer thresholdForDiagnosticsInMs = 60000;
```

Comment:
Isn't that way too high? I would think something around 3 seconds or so (maybe 10 to account for queries/FeedResponses with 1000 items( is a more realistic default threshold?

Reply:
We have retry policy upto 60 sec during 410s, if we bring it below that , there would be  to extra logging for the expected behavior . So the idea was to capture diagnostics for stuck requests (> 1 min) which bring the throughput to 0 for some time as we have experienced in past. Btw It is configurable , so we can change easily during run time without any extra effort

Reply:
@simplynaveen20  - I agree to @FabianMeiswinkel 's point here, why are we setting the threshold this high ?

Reply:
Thanks for providing the reasoning.